src/overlays/actors/ovl_Obj_Switch/z_obj_switch.c

```diff
@@ -384,7 +384,7 @@ void ObjSwitch_FloorUp(ObjSwitch* this, GlobalContext* globalCtx) {
             CollisionCheck_SetAC(globalCtx, &globalCtx->colChkCtx, &this->tris.col.base);
         }
     } else {
-        switch ((this->dyna.actor.params >> 4 & 7)) {
+        switch (this->dyna.actor.params >> 4 & 7) {
```

Comment:
These could do with () for disambiguation too, but they'll be replaced by macros soon enough anyway so I suppose it doesn't matter for now.

Reply:
Good point, but yeah there are a lot of expressions like these in the file so they should probably be cleaned up separately